showy.typ

```diff
@@ -124,22 +135,16 @@
             )
           )
       ]
-      v(-1.1em) // Avoid an inelegant space
     }
 
     /*
      * Body of the showybox
      */
     #block(
-      fill: frame.at("lower-color", default: white),
       width: 100%,
+      spacing: 0pt,
       inset:(x: 1em, y: 0.75em),
-      radius:
-        if title != "" {
-          (bottom: frame.at("radius", default: 5pt))
-        } else {
-          frame.at("radius", default: 5pt)
-        },
+      radius: frame.at("radius", default: 5pt),
```

Comment:
I've found that now this line of code is useless due to the `fill` removal in the body's `block`

```suggestion
```

Reply:
True, good find.